sections/model_overview/main.tex

```diff
@@ -1 +1,168 @@
-\section{Overview of game theoretic model}
\ No newline at end of file
+\section{Overview of game theoretic model}
+
+The problem studied is a 3-player game theoretic scenario between two queueing 
+systems and a service that distributes individuals to them. 
+Each player in this game has its own objectives and cares about meeting its own 
+targets.
+More specifically, the queueing systems' objective is captured by an upper bound
+of the time that a fixed proportion of individuals need to spend in the system, 
+while the distributor aims to minimise the time that its individuals 
+are blocked.  
+
+The queueing systems are designed in such a way where they can accept two types
+of individuals. 
+These are the individuals that the distributor allocates to them and 
+other individuals from other sources. 
+Each queueing system may then choose to block the individuals that arrive from 
+the distributor when the system reaches a certain capacity. 
+Thus, the set of strategies that they may choose to play in the game are defined 
+as the set of possible choices that the two queueing systems have. 
+In other words, each queueing system is given \( N_A \) and \( N_B \) number of 
+strategies where \( N_A \) and \( N_B \) are the total capacities of the two 
+queueing systems.
+The \textit{threshold} parameter is thus used to capture the chosen 
+strategy for each hospital where \( T_A \in \left[1, N_A \right] \) and 
+\( T_B \in \left[1, N_B \right] \)
+Their choice of strategy will then rely solely on satisfying their own 
+objective, which is to make sure that the waiting time of a proportion of 
+individuals will be below the predefined target time.
+
+\begin{equation}
+    P(W < \textit{ Target}) \geq \hat{P}
+\end{equation}
+
+where \(W\) is the waiting time of all individuals, \textit{Target} is the time 
```

Comment:
```suggestion
where \(W\) is the mean waiting time of all individuals, \textit{Target} is the time 
```

I'd not use `Target` in text at all (if you do it should not be in italics), just find some letter you haven't used yet and define that as the target.

Reply:
Hmmm🤔. Yeah, that makes sense. Will use `R` instead then 👍